dcc-storage-server/src/main/java/org/icgc/dcc/storage/server/service/upload/UploadStateStore.java

```diff
@@ -93,6 +96,15 @@
   @Autowired
   private BucketNamingService bucketNamingService;
 
+  @Data
+  @NoArgsConstructor
```

Comment:
I think `@NoArgsConstructor` is implied by `@Data`. In general, please prefer `@Value`. If you need to mutate, consider `@Wither`